fission/src/ui/panels/configuring/assembly-config/interfaces/ConfigureShotTrajectoryInterface.tsx

```diff
@@ -108,21 +108,17 @@ const ConfigureShotTrajectoryInterface: React.FC<ConfigEjectorProps> = ({ select
     // Not sure I like this, but made it a state and effect rather than a memo to add the cleanup to the end
     useEffect(() => {
         if (!selectedRobot?.ejectorPreferences) {
-            setTransformGizmo(undefined)
+            setEjectorMesh(undefined)
             return
         }
 
-        const gizmo = new TransformGizmos(
-            new THREE.Mesh(
-                new THREE.ConeGeometry(0.1, 0.4, 4).rotateX(Math.PI / 2.0).translate(0, 0, 0.2),
-                World.SceneRenderer.CreateToonMaterial(ReactRgbaColor_ThreeColor(theme.HighlightSelect.color))
-            )
+        const mesh = new THREE.Mesh(
+            new THREE.ConeGeometry(0.1, 0.4, 4).rotateX(Math.PI / 2.0).translate(0, 0, 0.2),
+            World.SceneRenderer.CreateToonMaterial(ReactRgbaColor_ThreeColor(theme.HighlightSelect.color))
         )
 
-        ;(gizmo.mesh.material as THREE.Material).depthTest = false
-        gizmo.AddMeshToScene()
-        gizmo.CreateGizmo("translate", 1.5)
-        gizmo.CreateGizmo("rotate", 2.0)
+        if (!mesh) return
+        ;(mesh.material as THREE.Material).depthTest = false
```

Comment:
```suggestion
        const material: THREE.Material = mesh.material
        material.depthTest = false
```

I like the look of this better but it doesn't really matter.